docs/language/operators/from.md

```diff
@@ -19,16 +21,26 @@ from (
 
 The `from` operator identifies one or more data sources and transmits
 their data to its output.  A data source can be
-* the name of a data pool in a Zed lake;
+* the name of a data pool in a Zed lake, optionally at a specific [branch or commit ID](../../commands/zed.md#142-commitish);
+* the names of multiple data pools, expressed as a [regular expression](../overview.md#711-regular-expressions) or [glob](../overview.md#712-globs) pattern;
 * a path to a file; or
 * an HTTP, HTTPS, or S3 URI.
 Paths and URIs may be followed by an optional format specifier.
 
-In the first three forms, a single source is connected to a single output.
-In the fourth form, multiple sources are accessed in parallel and may be
+Sourcing data from pools is only possible when querying a lake, such as
+via the [`zed` command](../../commands/zed.md) or
+[Zed lake API](../../lake/api.md). Sourcing data from files is only possible
+with the [`zq` command](../../commands/zq.md).
+
+When a single pool name is specified without `@`-referencing a commit or ID, or
+when using a pool pattern, the tip of the `main` branch of each pool is
+accessed.
+
+In the first four forms, a single source is connected to a single output.
+In the fifth form, multiple sources are accessed in parallel and may be
 [joined](join.md), [combined](combine.md), or [merged](merge.md).
 
-A data path can be split with the `fork` operator as in
+A data path can be split with the [fork operator](fork.md) as in
```

Comment:
This bit on `fork` and the parts that follow on `join` and `switch` feel semi-redundant with stuff in the Language Overview doc, so I thought about removing it all. However, since these all start out with `from` I figured they're worth keeping.